src/hidra/receiver/asapo_transfer.py

```diff
@@ -67,28 +70,11 @@ def stop(self):
 def main():
     parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter,
                                      description='Transfer files metadata to ASAPO')
-    parser.add_argument('--endpoint', type=str, help='ASAPO produces endpoint')
-    parser.add_argument('--beamtime', type=str, help='ASAPO produces beamtime')
-    parser.add_argument('--beamline', type=str, help='ASAPO produces beamline')
-    parser.add_argument('--default-data-source', type=str, help='ASAPO data_source')
-    parser.add_argument('--token', type=str, help='ASAPO produces token')
-    parser.add_argument('--n_threads', type=int, help='Number of threds for ASAPO producer',
-                        default=1)
-    parser.add_argument('--start_file_idx', type=int, help='Starting file index',
-                        default=1)
-    parser.add_argument('--file_regex', type=str, help='Template to file path, which includes `stream` and `file_idx`',
-                        default="current/raw/(?P<scan_id>.*)_(?P<file_idx_in_scan>.*).h5")
-    parser.add_argument('--timeout', type=float, help='ASAPO send timeout in [s]', default=30)
-    parser.add_argument('--signal_host', type=str, help='Signal host', default='localhost')
-    parser.add_argument('--target_host', type=str, help='Target host', default='localhost')
-    parser.add_argument('--target_port', type=str, help='Target port', default='50101')
-    parser.add_argument('--target_dir', type=str, help='Target directory', default='')
-    parser.add_argument('--detector_id', type=str, help='Detector hostname', default='')
-    parser.add_argument('--reconnect_timeout', type=int, help='Timeout to reconnect to sender',
-                        default=3)
-    parser.add_argument("--log-level", default="INFO", choices=["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"],
-                        help="Set log level for the application")
+    parser.add_argument('--config_file', type=str, help='Path to config file')
```

Comment:
It might be safer to take a path to a directory instead and look for a file named `asapo_transfer_{identifier}.yaml` within that directory. This way it is impossible to give a wrong combination of config file and identifier. On the other hand, it might get more difficult to create a test config. What do you think?